aws/credentials/ssocreds/sso_cached_token.go

```diff
@@ -0,0 +1,229 @@
+package ssocreds
+
+import (
+	"crypto/sha1"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"github.com/aws/aws-sdk-go/internal/shareddefaults"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var osUserHomeDur = shareddefaults.UserHomeDir
```

Comment:
i get what youre doing here: you seem to be saving it out to a variable so you can inject over it in the test case. but id bias towards descriptive difference in naming rather than intentional misspelling. something like `resolvedOsUserHomeDir` or `injectableOsUserHomeDir`